micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/Configuration.java

```diff
@@ -162,6 +163,215 @@ public void setProperties(@MapFormat(transformation = MapFormat.MapTransformatio
          */
         // Close #219 - This method is needed so that the auto-completion of properties works
         boolean isIndexRedirectEnabled();
+
+        @NotNull
+        HeaderSecurity getHeadersecurity();
+
+        @ConfigurationProperties("header-security")
+        interface HeaderSecurity {
+
+            /**
+             * The header can be entirely disabled if set to true. Default is false.
+             *
+             * @return if xss protection is disabled
+             */
+            @Bindable(defaultValue = "false")
+            boolean isXssProtectionDisabled();
+
+            /**
+             * The allowed set of values:
+             * <ul>
+             *     <li>BLOCK: If the browser detects a cross-site scripting attack, the page is blocked completely</li>
+             *     <li>SANITIZE: If the browser detects a cross-site scripting attack, the page is sanitized from suspicious parts (value 0)</li>
+             * </ul>
+             * Default is BLOCK. Note:
+             * <ul>
+             *     <li>Is ignored when .xss-protection-disabled is set to true</li>
+             *     <li>Cannot be set in conjunction with .xss-protection-value</li>
+             * </ul>
+             *
+             * @return the xss protection option
+             */
+            Optional<String> getXssProtectionOption();
+
+            /**
+             * A custom value for the header can be specified. Default is '1; mode=block'
+             * <ul>
+             *     <li>Is ignored when .xss-protection-disabled is set to true</li>
```

Comment:
I'd remove the dot before xss-protection-disabled here - and in all other JavaDocs in this class.